lava-v2-jobs-from-api.py

```diff
@@ -71,23 +72,79 @@ def get_builds(api, token, config):
     return builds
 
 
-def add_callback_params(params, config, plan):
-    callback = config.get('callback')
-    if not callback:
-        return
+def is_callbacks_file_valid(callbacks):
+    expected_fields = ['type', 'token', 'url', 'dataset', 'method', 'content-type']
+    broken_callbacks = []
+
+    for c in callbacks:
+        if not c.get('url'):
+            print('Warning: one of the callbacks has no "url" set, removing it from the list')
```

Comment:
logging sounds a better fit, but this is a case for another patch.

Reply:
Agreed, I could propose some refactoring later on